cip/CIP2016-07-07-Parameter-syntax.adoc

```diff
@@ -0,0 +1,91 @@
+= CIP2016-07-07 - Parameter syntax
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Mats Rydberg <mats@neotechnology.com>
+
+[abstract]
+.Abstract
+--
+This CIP suggests changing syntax for parameter usage, from curly braces (`{}`) to dollars (`$`).
+Curly brackets are used by the language for literal maps and property predicates, and this overloading may cause confusion in usage.
+Using a different, unique character causes parameters to be more visible for the human eye, and by using only a single character for the parameter instead of one opening and one closing, the syntax becomes somewhat cleaner.
+--
+
+toc::[]
+
+
+== Motivation
+
+//Provide the rationale and justification for the proposed feature(s).
+
+The clause `MATCH (a {prop: {param}})` contains a nested curly brace structure, which is not ideal for human readability.
+The alternative `MATCH (a {prop: $param})` does a much better job at signalling the parameter usage to the reader.
```

Comment:
I would suggest rewording
'does a much better job at signalling the parameter usage to the reader.' 
to something like:
'makes the parameter usage far more legible.'